relay/examples/publish.rs

```diff
@@ -35,8 +32,8 @@ async fn main() {
     let request = CallbackRequest {
         callback_contract: args.address,
         // you can use the command `solc --hashes contracts/BonsaiStarter.sol`
-        // to get the value for your actual contract
-        function_selector: [0xe8, 0x08, 0x02, 0xa2],
+        // to get the value for your actual contract (9f2275c0: storeResult(uint256,uint256))
+        function_selector: [0x9f, 0x22, 0x75, 0xc0],
```

Comment:
Non blocking: but is it possible to use `forge bind` or `ethers_solc` to automatically generate the function selector at build time from the .sol files?
Maybe something in: 
https://github.com/foundry-rs/foundry-rust-template#generating-rust-bindings-to-the-contracts / https://book.getfoundry.sh/reference/forge/forge-bind

Or in the output of ethers_solc would have it accessible.